File: Tigran/Week 1/cash.c
#include <stdio.h>
#include <cs50.h>
int main()
{
    
    float cash = -1;
    while (cash <0)
    {
        cash = get_float("Enter cash : ");    
    }
     float fraction = cash - ((long)cash);
     fraction = fraction*100;
    
     int cash1 = ((int)fraction);
     int count = cash1/25 + (cash1%25)/10 + ((cash1%25)%10)/5 + ((cash1%25)%10)%5;
    
     printf("%d \n",count);

    return 0;    
}

File: Harut/workspace/pset6/cash/cash.py
from cs50 import get_float

while True:
    d = get_float("$ = ")
    if (d > 0):
        break

c = d * 100
q = c // 25 + c % 25 // 10 + c % 25 % 10 // 5 + c % 25 % 10 % 5 // 1
print(q)